### DISCUSSION TITLE
pgvector enhancement called pgvectorscale

### DISCUSSION THREAD

**Discussion by JungeWerther** [supabase/supabase#29095] [Feature Requests]
Hi! I came across [this project](https://github.com/timescale/pgvectorscale) that extends pgvector. Would make Supabase **very attractive** for RAG projects.

**Comment by timothyerwin** [comment]
yeah looking for this...don't see it in supabase

**Comment by arthikek** [comment]
I have the same problem..

**Comment by arthikek** [comment]
I am going to try to add it with a PR hahah
